static/templates/index.html

```diff
@@ -10,6 +10,13 @@
       href="{{root}}/skin/index.css?KIWIXCACHEID"
       rel="Stylesheet"
     />
+    <link 
+      rel="alternate" 
+      type="application/rss+xml" 
+      title="Kiwix RSS Feed"
+      id="rssExtension" 
+      href="/catalog/v2/entries"
```

Comment:
Missing `{{root}}`. I know that it doesn't matter since it will be fixed by `updateRssFeedLink()` after the HTML is loaded, but it is better to be consistent.